federation/pkg/federatedtypes/hpa.go

```diff
@@ -36,26 +36,34 @@ import (
 const (
 	HpaKind           = "horizontalpodautoscaler"
 	HpaControllerName = "horizontalpodautoscalers"
-	// This is a tunable which does not change replica nums
-	// on an existing local hpa, before this timeout, if it
-	// did scale already (avoids thrashing of replicas around).
-	scaleForbiddenWindow = 5 * time.Minute
 	// This is used as the default min for hpa object submitted
 	// to federation, in a situation where the default is for
 	// some reason not present (Spec.MinReplicas == nil)
 	hpaMinReplicaDefault = int32(1)
+	ScaleForbiddneWindow = 2 * time.Minute
```

Comment:
Actually, I did add quite an elaborate comment on the new argument, thus dropper the one from here; but I see that it makes sense to keep the comment here also. Updated.